### PR TITLE
Further customization to logger

### PR DESCRIPTION
**Pull request by abdalla-elmahdy**
- Made seperate loggers for dev and production environmets to be customized in the future for each environment if needed
- Removed trace field, cookie and authorization headers from meta data logging

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -23,7 +23,9 @@ app.use(
     winstonInstance: logger,
     meta: false,
     msg: 'HTTP {{req.method}} {{req.url}}',
-    expressFormat: true
+    expressFormat: true,
+    blacklistedMetaFields: ['trace'],
+    headerBlacklist: ['authorization', 'cookie']
```
Comment (saifaldin) [inline app.js]: Great work, please consider adding these values as config values

File: app.js [hunk 2]
```diff
@@ -45,7 +47,9 @@ app.use('/images', imageRouter);
 
 app.use(
   expressWinston.errorLogger({
-    winstonInstance: logger
+    winstonInstance: logger,
+    blacklistedMetaFields: ['trace'],
+    headerBlacklist: ['authorization', 'cookie']
```
Comment (saifaldin) [inline app.js]: These too as config values

File: util/logger.js [hunk 3]
```diff
@@ -1,6 +1,27 @@
...
+  transports: [
+    new winston.transports.Console({ colorize: true }),
+    new winston.transports.File({
+      filename: './logs/error.log',
+      level: 'error',
+      maxsize: '100000000' //100 mb in bytes
+    }),
+    new winston.transports.File({
+      filename: './logs/combined.log',
+      maxsize: '100000000' //100 mb in bytes
```
Comment (saifaldin) [inline util/logger.js]: Also the log file max size as a config value

File: app.js (hunk 1 above)
Comment (abdalla-elmahdy) [reply app.js]: The blacklisted headers hold sensitive information, I don't think they should be logged regardless of the environment.

Comment (saifaldin) [reply app.js]: then add it as a default config value

File: util/logger.js (hunk 3 above)
Comment (abdalla-elmahdy) [reply util/logger.js]: done

**Review by saifaldin — APPROVED** [review]
LGTM

**Pull request merged**